packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -199,7 +269,7 @@ export default defineComponent({
 						if (newAuth) {
 							const credentialType = getNodeCredentialForSelectedAuthType(
 								nodeType,
-								newAuth.toString(),
+								JSON.stringify(newAuth),
```

Comment:
Does this still work correctly? `.toString()` and `JSON.stringify` produce different values for some types. Eg.

```ts
"test".toString() // 'test'
JSON.stringify("test") // '"test"'
```